packages/devextreme/testing/tests/DevExpress.ui.widgets.dataGrid/selection.tests.js

```diff
@@ -4643,6 +4643,33 @@ QUnit.module('Deferred selection', {
         // assert
         assert.deepEqual(this.option('selectionFilter'), []);
     });
+
+    QUnit.test('DataGrid - getSelectedRowKeys method returns incorrect keys if deferred selection.sensitivity is "variant" (T1280037)', function(assert) {
```

Comment:
Almost fixed :)
'DataGrid - getSelectedRowKeys method **returns incorrect keys** if deferred selection is enabled (T1280037)' -> 'DataGrid - getSelectedRowKeys method **should return the correct keys** if selection.sensitivity is 'variant' (T1280037)'